tests/foreman/cli/test_import.py

```diff
@@ -56,7 +74,7 @@ def test_import_orgs_default(self):
         orgs = set(org['name'] for org in Org.list().stdout)
         imporgs = set(
             org['organization'] for
-            org in csv_to_dataset(ssh_cat.stdout)
```

Comment:
According to this change, the `ssh_import` and `ssh_cat` variables are no longer used. Drop them from this method.